CHANGELOG.md

```diff
@@ -9,12 +9,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
+- Tarantool 3.0 role for expirationd.
+
 ### Changed
 
 - Updated the 'space_index_test.lua' to drop and recreate the test space
   atomically. This prevents the space access failure in the expirationd
   task fiber if the `space:drop` function is transactional (#157).
 
+- Updated version of `luatest` in `make deps` to 1.0.1 to support tarantool 3.0
```

Comment:
No need in the newline here. Just in the next line:

```
- foo (#157).
- bar (#161).
```

Reply:
Changed